tests/transactional/enable_selinux.pm

```diff
@@ -34,8 +51,9 @@ sub run {
 
     # install and enable SELinux if not done by default
     if (script_run('test -d /sys/fs/selinux && test -e /etc/selinux/config && getenforce | grep -i "enforcing" >/dev/null') != 0) {
+        die("SELinux should be enabled by default on SLEM 15.4+") if (is_sle_micro('5.4+') || is_leap_micro('5.4+'));
         trup_call('setup-selinux');
-        upload_logs('/var/log/transactional-update.log');
+        upload_logs('/var/log/transactional-update.txt');
```

Comment:
> seems this is causing failures in your VRs, why are you changin the extension?

I'm changing the extension to make it more browser-friendly since the `.log` extensions always downloads it and `.txt` just opens it in a browser.